java/modules/commons/src/main/java/org/apache/synapse/commons/evaluators/EvaluatorContext.java

```diff
@@ -26,13 +26,14 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.checkerframework.checker.startswith.qual.*;
 /**
  * Holds the information about the HTTP request. Created on per request basis and
```

Comment:
That this class is specifically labeled as containing an HTTP request makes it seem unlikely that its `url` field actually obeys the `@StartsWith` annotation on it. Perhaps the documentation is wrong?

Where is this class used?